src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <span class="logo__container">
+          <a class="logo__link"
+             href="index.html">
+            <img class="logo__image"
+                 src="images/logo.png"
+                 alt="lo">
```

Comment:
Check the [Checklist №13](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) on long multiline tag. This should be fixed on entire document